### ISSUE TITLE
reverse, shuffle In and Out type not working

### ISSUE THREAD

**Issue by mrkenng** [gitWhatever/textillate-vanilla#1]
Good vanilla version of Textillate. However I ran into an error when trying to set reverse: true on both in and out. 

```
Uncaught TypeError: Failed to set an indexed property on 'HTMLCollection': Index property setter is not supported.
    at reverse (index.js:381)
    at animateTokens (index.js:398)
    at MTextillate._in [as in] (index.js:591)
    at run (index.js:498)
    at index.js:519
```

**Comment by gitWhatever** [comment]
Thanks for using and feedback, I will repair it as soon as possible

**Comment by gitWhatever** [comment]
Thanks for using and feedback, I will repair it as soon as possible

**Comment by gitWhatever** [comment]
Problem has been solved, the latest npm version is 1.0.1。
